### DISCUSSION TITLE
Add a word filter to osu!lazer public lobbies names

### DISCUSSION THREAD

**Discussion by goldengun111** [ppy/osu#16246] [Ideas] [answered]
The title explains everything, there should be a word filter so things like [this ](https://imgur.com/a/SiYDleZ)don't happen. that's about it.

The people in the lobby are: flarmax12, IMABEEFBOYY, jaavaitlits, all 3 of them are from Estonia. suggest banning them

**Comment by Theighlin** [comment, marked as answer]
This has been happening for a while though. And if a bad word filter is put in place, players will resort to using their native language, shoud osu filter every word that is bad in some language or another?

People will always find clever ways to avoid censorship, like using different characters or saying a word backwards. Imo these cases should be dealt with on a per user basis, since they can't multi account. They should be restricted temporarily for doing it once, and permanently if they are caught doing it again. Isn't this also how chat is managed right now? They should know the rules of the game they are playing.

**Reply by goldengun111** [reply]
True but anything would be better than nothing at this point, filtering every language is basically impossible, and it's obvious that people will try to avoid it. Cases like these should be a user to user basis but he issue is that they aren't being dealt with in the first place, I've only been playing lazer for maybe 2 months so I haven't seen a lot of things like that but the issue is that anyone can make a lobby with such a name with no immediate repercussions, this is most likely because lazer isn't the main client for people and it has a fraction of the player base compared to regular osu!. Still even a barebones word filter would be better than nothing at least until lazer becomes the main client.

**Reply by peppy** [reply]
Please report the users via their profile. We restrict or silence for such behaviour.
